lib/logstash/filters/grok/timeout_enforcer.rb

```diff
@@ -19,19 +19,15 @@ def grok_till_timeout(grok, field, value)
       grok.execute(value)
     rescue InterruptedRegexpError => e
       raise ::LogStash::Filters::Grok::TimeoutException.new(grok, field, value)
+    rescue java.lang.InterruptedException => e
+      # NOOP, we don't expect these, but maybe some interruptible thing could be
+      # added to grok besides regexps
+      @logger.debug("Unexpected interruptible caught during grok. This isn't a problem most likely")
```

Comment:
since the grok has likely been interrupted, shouldn't we either log at a higher, warn or error or bubble up the exception?

Reply:
Hmmm, good point, yes, it should be a warn.